src/data/finsupp.lean

```diff
@@ -185,12 +216,14 @@ end
 
 end single
 
+/-! ### Declarations about `on_finset` -/
+
 section on_finset
 variables [has_zero β]
 
-/-- `on_finset s f hf` is the finsupp function representing `f` restricted to the set `s`.
-The function needs to be 0 outside of `s`. Use this when the set needs filtered anyway, otherwise
-often better set representation is available. -/
+/-- `on_finset s f hf` is the finsupp function representing `f` restricted to the finset `s`.
+  The function needs to be `0` outside of `s`. Use this when the set needs to be filtered anyways,
```

Comment:
"needs filtered" is a lovely example of Pennsylvania dialect ;) (I agree with the change!)